### PR TITLE
Master

### PR DESCRIPTION
**Pull request by CryptoDeveloper87**
add BlocX Network

### CODE REVIEW

**Comment by z3r086** [comment]
> add BlocX Network

**Review by CryptoDeveloper87 — COMMENTED** [review]
Update BlocX Network

**Review by Akaslove99 — APPROVED** [review]

**Comment by CryptoDeveloper87** [comment]
Why does the build keep failing.. even with out my update..

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Review by CryptoDeveloper87 — COMMENTED** [review]
Update BlocX Network

**Comment by CryptoDeveloper87** [comment]
> Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json) e.g. run `npx prettier --write _data/*/*.json`

I have done this , Thank you

File: _data/chains/eip155-879151.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "infoURL": "https://blocxchain.org",
+  "shortName": "blx",
+  "chainId": 879151,
+  "networkId": 879151,
+  "icon": "blocx",
+  "explorers": [
+    {
+      "name": "blxscan",
+      "url": "https://explorer.blxscan.com/",
```
Comment (ligi) [inline _data/chains/eip155-879151.json:19]:
```suggestion
      "url": "https://explorer.blxscan.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]
Do not checkin any vscode stuff

I could not resolve your CID with `ipfs get` in a reasonable time - and before having a back and forth here I require you to put it in iconDownload. Before this back and forth did cost me a lot of time that I do not have.

**Pull request closed without merging**